tw-tasks-core/src/main/java/com/transferwise/tasks/helpers/kafka/AdminClientTopicPartitionsManager.java

```diff
@@ -31,7 +33,9 @@
   @Override
   public void setPartitionsCount(String topic, int partitionsCount) {
     ExceptionUtils.doUnchecked(() -> {
-      AdminClient adminClient = AdminClient.create(kafkaConfiguration.getKafkaProperties().buildAdminProperties());
+      Map<String, Object> adminConfig = kafkaConfiguration.getKafkaProperties().buildAdminProperties();
+      adminConfig.put(AdminClientConfig.CLIENT_ID_CONFIG, "");
```

Comment:
Please add comment here for purpose. Even after reading PR description, this looks pretty obscure. And future readers likely won't take the time to dig up & read the PR desc.

Could also point to the id-generator utility, to which this seemingly has some relation?

Reply:
added comments to make things clear.